PyEMD/tests/test_significancetest.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import unittest
+from PyEMD.significancetest import *
```

Comment:
For local development it can be helpful to have `from XX import *` but for release packages this is strongly discouraged. Please be explicit with what you import.

Reply:
done